Cannot find module 'path-to-regexp'

@dirkwa
This commit introduced `path-to-regexp` but `package.json`is missing it, so SignalK is broken now. Just wondering how this commit was verified?
[feat(plugins): allow plugins to declare route permission levels](https://github.com/SignalK/signalk-server/commit/8389b6fa29ff91df9d07400c4339beb8044cfc9a#diff-cd8c919d3b31d83aea6feea0dfc9dfb1d9cbd9a4c452638fb5ec0b6e9d01b78f) 

```
node:internal/modules/cjs/loader:1520
  throw err;
  ^

Error: Cannot find module 'path-to-regexp'
Require stack:
- /home/node/signalk/node_modules/signalk-server/dist/tokensecurity.js
- /home/node/signalk/node_modules/signalk-server/dist/security.js
- /home/node/signalk/node_modules/signalk-server/dist/index.js
- /home/node/signalk/node_modules/signalk-server/bin/signalk-server
    at Module.<anonymous> (node:internal/modules/cjs/loader:1517:15)
    at ModuleInternal._resolveFilename (/home/node/signalk/node_modules/signalk-server/dist/baconjs-compat.js:68:32)
    at wrapResolveFilename (node:internal/modules/cjs/loader:1071:27)
    at defaultResolveImplForCJSLoading (node:internal/modules/cjs/loader:1095:10)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1122:12)
    at Module._load (node:internal/modules/cjs/loader:1294:5)
    at wrapModuleLoad (node:internal/modules/cjs/loader:255:19)
    at Module.require (node:internal/modules/cjs/loader:1617:12)
    at require (node:internal/modules/helpers:153:16)
    at Object.<anonymous> (/home/node/signalk/node_modules/signalk-server/dist/tokensecurity.js:29:22) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/node/signalk/node_modules/signalk-server/dist/tokensecurity.js',
    '/home/node/signalk/node_modules/signalk-server/dist/security.js',
    '/home/node/signalk/node_modules/signalk-server/dist/index.js',
    '/home/node/signalk/node_modules/signalk-server/bin/signalk-server'
  ]
}

Node.js v24.18.0
```

--- a/package.json
+++ b/package.json
@@ -120,6 +120,7 @@
     "ncp": "^2.0.0",
     "openid-client": "^6.8.4",
     "ora": "^5.4.1",
+    "path-to-regexp": "^0.1.12",
     "primus": "^7.3.5",
     "prompts": "^2.4.2",
     "rehype-parse": "^9.0.1",

